### PR TITLE
[NEXUS-4616] Send times as longs and display them based using user timezone

### PR DESCRIPTION
**Pull request by adreghiciu**

### CODE REVIEW

**Comment by cstamas** [comment]
+1, nice

File: nexus/nexus-test-harness/nexus-test-harness-its/src/test/java/org/sonatype/nexus/integrationtests/nexus2797/Nexus2797Validate140TaskUpgradeIT.java [hunk 1]
```diff
@@ -18,12 +18,14 @@
  */
 package org.sonatype.nexus.integrationtests.nexus2797;
 
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertNull;
```
Comment (velo) [inline nexus/nexus-test-harness/nexus-test-harness-its/src/test/java/org/sonatype/nexus/integrationtests/nexus2797/Nexus2797Validate140TaskUpgradeIT.java:22]: I don't have strong feelings about it, but we should be using Hamcrest assertThat() over regular assertions...

**Comment by velo** [comment]
+1

File: nexus/nexus-test-harness/nexus-test-harness-its/src/test/java/org/sonatype/nexus/integrationtests/nexus2797/Nexus2797Validate140TaskUpgradeIT.java (hunk 1 above)
Comment (adreghiciu) [inline nexus/nexus-test-harness/nexus-test-harness-its/src/test/java/org/sonatype/nexus/integrationtests/nexus2797/Nexus2797Validate140TaskUpgradeIT.java:22]: Right. Yet I did not interfere with test. Just adapted.

**Pull request merged**